packages/components/src/components/ScalarIcon/.svglintrc.js

```diff
@@ -0,0 +1,101 @@
+const fillOpts = {
+  'fill?': true,
+  'fill-rule?': true,
+  'clip-rule?': true,
+}
+
+const strokeOpts = {
+  'stroke?': true,
+  'stroke-linecap?': true,
+  'stroke-linejoin?': true,
+}
+
+export default {
+  rules: {
+    elm: {
+      // Only allow one <svg> node
+      'svg': 1,
+      // Allow up to one <title>
+      'title': [0, 1],
+      // Allow shape elements
+      'path': [0, 999],
+      'rect': [0, 999],
+      'circle': [0, 999],
+      // Don't allow other elements
+      '*': false,
+    },
+    attr: [
+      {
+        // Ensure that the SVG element has the appropriate attributes
+        'rule::selector': 'svg',
+        // Don't allow extraneous attributes
+        'rule::whitelist': true,
+        // ViewBox must be square and namespace must be set
+        'viewBox': /^0 0 (\d+) \1$/,
```

Comment:
should we set a specific viewbox size at some point?

Reply:
Yeah I think so, I just didn't want to reformat all the icons as part of this PR. Currently I have 12px set up as the standard in the readme since line icons with a stroke width of 1 look about right at that size.